ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -64,9 +65,9 @@
   @Option(
       names = {"--downstream-http-request-timeout"},
       description =
-          "Timeout (in milliseconds) to wait for downstream request to timeout (default: ${DEFAULT-VALUE})",
+          "Timeout in seconds to wait for downstream request to timeout (default: ${DEFAULT-VALUE})",
```

Comment:
contains "timeout" twice ... can this be reworded?

Reply:
remove last timeout think it's superfluous. done